.dotnet/api/OpenAI.netstandard2.0.cs

```diff
@@ -2876,39 +2895,12 @@ public class RealtimeConversationClient {
-        protected internal RealtimeConversationSession(RealtimeConversationClient parentClient, Uri endpoint, ApiKeyCredential credential);
-        public Task AddItemAsync(ConversationItem item, string previousItemId, CancellationToken cancellationToken = default);
-        public Task AddItemAsync(ConversationItem item, CancellationToken cancellationToken = default);
-        public Task CancelResponseTurnAsync(CancellationToken cancellationToken = default);
-        public Task CommitPendingAudioAsync(CancellationToken cancellationToken = default);
-        public Task ConfigureSessionAsync(ConversationSessionOptions sessionOptions, CancellationToken cancellationToken = default);
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        protected internal virtual void Connect(RequestOptions options);
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        protected internal virtual Task ConnectAsync(RequestOptions options);
-        public Task DeleteItemAsync(string itemId, CancellationToken cancellationToken = default);
-        public void Dispose();
-        public Task InterruptTurnAsync(CancellationToken cancellationToken = default);
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public virtual IEnumerable<ClientResult> ReceiveUpdates(RequestOptions options);
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public virtual IAsyncEnumerable<ClientResult> ReceiveUpdatesAsync(RequestOptions options);
-        public IAsyncEnumerable<ConversationUpdate> ReceiveUpdatesAsync(CancellationToken cancellationToken = default);
-        public Task SendAudioAsync(BinaryData audio, CancellationToken cancellationToken = default);
-        public Task SendAudioAsync(Stream audio, CancellationToken cancellationToken = default);
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public virtual void SendCommand(BinaryData data, RequestOptions options);
-        [EditorBrowsable(EditorBrowsableState.Never)]
-        public virtual Task SendCommandAsync(BinaryData data, RequestOptions options);
-        public Task StartResponseTurnAsync(CancellationToken cancellationToken = default);
+        public virtual AssistantConversation StartConversation(AssistantConversationOptions options, CancellationToken cancellationToken = default);
+        public virtual AssistantConversation StartConversation(BinaryContent configuration, TwoWayClient.TwoWayPipelineOptions conversationOptions, RequestOptions requestOptions);
+        public AssistantConversation StartConversation(CancellationToken cancellationToken = default);
+        public virtual Task<AssistantConversation> StartConversationAsync(AssistantConversationOptions options, CancellationToken cancellationToken = default);
+        public virtual Task<AssistantConversation> StartConversationAsync(BinaryContent configuration, TwoWayClient.TwoWayPipelineOptions conversationOptions, RequestOptions requestOptions);
```

Comment:
What does it look like to configure the conversation in a protocol-only world?  (When we have munged HTTP and WebSocket messages -- i.e. making two calls on different protocols from the Start method?)